### DISCUSSION TITLE
Deploying on Digital Ocean vs Laravel Forge

### DISCUSSION THREAD

**Discussion by jayrdi** [statamic/cms#8884] [Q&A] [answered]
Hi, I've completed by Statamic site locally and am looking to deploy live. It's already connected to a private GitHub account.

I've never done this before, so looking for the easiest solution. I was originally going for a Digital Ocean App Platform, but have seen Laravel Forge mentioned as a potentially easier way to do it in the Statamic documentation.

What are the pros and cons of these two options? Do they cost about the same? Which would you recommend as a solution for someone doing this for the first time?

Thanks for your help.

Edit: I've realised after more research that laravel forge is a layer between digital ocean to help to deploy apps. I guess then, you have to pay for both DO and LF? I suppose my amended question then is what the advantages are to using LF? Would it be more suitable for someone like me who is new to server mgmt and deploying sites?

**Comment by ryanmitchell** [comment]
Forge (and Ploi) take the hassle out of the server maintenance and deploy process for you. You can certainly deploy without them (using deployer, envoyer) but to be honest I wouldn't - they make it so simple.

**Reply by jayrdi** [reply]
Yes, I ended up using Ploi after seeing recommendations and finding it great to use

**Comment by jasonvarga** [comment, marked as answer]
There is a difference between Digital Ocean itself (the "Droplets" are the servers), and Digital Ocean App Platform which is their version of server management tool like Forge.

I've personally not used Digital Ocean App Platform, so I'll leave it for someone else to chime in there, but in my limited knowledge it looks to be more complicated than Forge. I believe it's more of a "serverless" thing which tend to be more complicated.

I think on App Platform you end up needing to pay for more things as you need them. If you want a database, you gotta add that to the price. Queue worker, add more. etc.

With Forge, you pay for Forge, then you pay the monthly price for a Digital Ocean droplet. You can do what you want with the server (and Forge makes that part easy). If you want a database, Forge can add one to the server. Queue worker, sure. You don't need to keep adding costs.

**Reply by jayrdi** [reply]
Thanks for your help. In the end I took the advice of Stoffelio on Reddit and opted for Ploi and Hetzner. Got the site running and happy with the setup 👍
